DEMO.md

```diff
@@ -73,10 +73,12 @@ the response should look like
   "href":"/api/v1/bridges/87508471-ee0f-4f53-b574-da8a61285986",
   "submitted_at":"2021-09-24T11:29:33.086649+0000",
   "status":"AVAILABLE",
-  "endpoint":"/ingress/events/87508471-ee0f-4f53-b574-da8a61285986"
+  "endpoint":"http://ob-87508471-ee0f-4f53-b574-da8a61285986-ob-kekkobar.apps.openbridge-dev.fdvn.p1.openshiftapps.com/"
```

Comment:
@r00ta Unrelated to this PR, but does the Ingress support plain HTTP? Pretty sure we should only support HTTPS connections to the ingress.

Reply:
@robpblake yes we should. We still have to configure the cluster properly

Reply:
https://issues.redhat.com/browse/MGDOBR-183

Reply:
Thanks @r00ta . AFAICR our demo cluster is using LetsEncrypt for SSL certs, but something for us to check.